RxCocoa/RxCocoa/iOS/UITextField+Rx.swift

```diff
@@ -12,21 +12,31 @@ import UIKit
 
 extension UITextField {
     public func rx_text() -> Observable<String> {
+        return observableForControlEvent(.EditingChanged)
+    }
+
+    public func rx_endOnExit() -> Observable<String> {
```

Comment:
I can go either way with it. I named it `endOnExit` because you used `text` for `EditingChanged` so I assumed you were trying to keep function names terse. Just let me know.